files/build_templates/init_cfg.json.j2

```diff
@@ -10,7 +10,7 @@
             "polling_interval": "300",
 {%- for crm_res in ["ipv4_route", "ipv6_route", "ipv4_nexthop", "ipv6_nexthop", "ipv4_neighbor",
                     "ipv6_neighbor", "nexthop_group_member", "nexthop_group", "acl_table",
-                    "acl_group", "acl_entry", "acl_counter", "fdb_entry"] %}
+                    "acl_group", "acl_entry", "acl_counter", "fdb_entry", "snat_entry", "dnat_entry", "ipmc_entry"] %}
```

Comment:
what if snat_entry, dnat, ipmc_entry not enabled in the image? will there be any issue to have crm threshold there?

Reply:
Orch will ignore these new entries on the config_db.json with a log message like the one given below.

Nov 24 15:56:48.149727 sonic ERR swss#orchagent: :- handleSetCommand: Failed to parse CRM Config configuration. Unknown attribute snat_entry_low_threshold.